ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -81,7 +82,7 @@ const BaseFeeInput = () => {
 
   const updateBaseFee = useCallback(
     (value) => {
-      setBaseFee(value);
+      setBaseFee(new Numeric(value, 10)?.toString());
```

Comment:
Is the core fix here that `toString` from `Numeric` doesn't use scientific notation?

Reply:
Yes, converting using `Numeric` properly saves as `string`. 
The underlying issue is that the `value` is coming from the `FormField` component as `number` and it saves as `string` with scientific notation (when using more than 6 decimal places) as a result it gets stuck the next time you try to send a transaction.

Ideally, it should always be `string`, I will open a ticket apart, maybe refactor the component or use `FormTextField` with type `string`, if necessary preventing non-decimals, but all of it will also involve layout changes.